files/commonfiles/class/Utility.php

```
@@ -151,13 +151,16 @@ public static function truncateHtml($text, $length = 100, $ending = '...', $exac
     /**
-     * @param \Xmf\Module\Helper $helper
-     * @param array|null         $options
+     * @param \Xmf\Module\Helper|null $helper
+     * @param array|null $options
      * @return \XoopsFormDhtmlTextArea|\XoopsFormEditor
      */
-    public static function getEditor($helper = null, $options = null)
+    public static function getEditor(\Xmf\Module\Helper $helper = null, array $options = null)
     {
         /** @var Modulebuilder\Helper $helper */
+        if (null === $helper) {
```

PHPStan (2.1.46)

[error] PHPDoc tag `@var` with type XoopsModules\Modulebuilder\Helper is not subtype of native type Xmf\Module\Helper|null.


(varTag.nativeType)